Reserves: clarify NFTs, simplify some stuff (#1708)
XRPLF/xrpl-dev-portal: content/concepts/payment-system-basics/accounts/reserves.md

To have an account, an address must hold a minimum amount of XRP in the shared global ledger. To fund a new address, you must receive enough XRP at that address to meet the reserve requirement. You cannot send the reserved XRP to others, but you can recover some of the XRP by [deleting the account](accounts.html#deletion-of-accounts).

The reserve requirement changes from time to time due to the [Fee Voting](fee-voting.html) process, where validators can agree to new reserve settings.

## Base Reserve and Owner Reserve

The reserve requirement has two parts:

* The **Base Reserve** is a minimum amount of XRP that is required for each address in the ledger.
* The **Owner Reserve** is an increase to the reserve requirement for each object that the address owns in the ledger. The cost per item is also called the _incremental reserve_.

The current reserve requirements on Mainnet are:

- Base reserve: **10 XRP**
- Owner reserve: **2 XRP** per item

Reserves on other networks may vary.

## Owner Reserves

> Many objects in the ledger (ledger entries) are owned by a particular account. Usually, the owner is the account that created the object. Each object increases the owner's total reserve requirement by the owner reserve. When objects are removed from the ledger, they no longer count against the reserve requirement.

Reviewer: ```suggestion
Many objects in the ledger (ledger entries) are owned by an account--usually the account that created it. Each object owned by an account increases their "owner reserve" requirement. When objects are removed from the ledger, they no longer count against the reserve requirement.
```
When objects are removed does the reserve XRP get added back to the account balance?

Author: The balance never changes, but the amount that you're allowed to spend from it does.

> Objects that count towards their owner's reserve requirement include: [Checks](checks.html), [Deposit Preauthorizations](depositauth.html#preauthorization), [Escrows](escrow.html), [NFT Offers](non-fungible-token-transfers.html), [NFT Pages](non-fungible-tokens.html), [Offers](offer.html), [Payment Channels](payment-channels.html), [Signer Lists](multi-signing.html), [Tickets](tickets.html), and [Trust Lines](trust-lines-and-issuing.html).

Reviewer: ```suggestion
Objects that count towards their owner's reserve requirement include:
- [Checks](checks.html)
- [Deposit Preauthorizations](depositauth.html#preauthorization)
- [Escrows](escrow.html), [NFT Offers](non-fungible-token-transfers.html)
- [NFT Pages](non-fungible-tokens.html)
- [Offers](offer.html)
- [Payment Channels](payment-channels.html)
- [Signer Lists](multi-signing.html)
- [Tickets](tickets.html)
- [Trust Lines](trust-lines-and-issuing.html).
```

> Some special cases:

Reviewer: It might be easier to include each special case as a note in the corresponding bullet point above.

Author: That's more or less how it was before this change, and it was an overwhelming amount of info and hard to identify which ones are special cases and which aren't. My goal here is to make the full list more compact (so you can scroll over it more easily) and then only have bullet points for the exceptions.

- Non-Fungible Tokens (NFTs) are grouped into pages containing up to 32 NFTs each, and the owner reserve applies per page rather than per NFT. Due to the mechanism for splitting and combining pages, the number of NFTs actually stored per page varies. See also: [Reserve for NFTokenPage objects](nftokenpage.html#reserve-for-nftokenpage-objects).
- Trust lines (`RippleState` entries) are shared between two accounts. The owner reserve can apply to one or both of them. Most often, the token holder owes a reserve and the issuer does not. See also: [RippleState: Contributing to the Owner Reserve](ripplestate.html#contributing-to-the-owner-reserve).
- Signer lists created before the [MultiSignReserve amendment][] activated in April 2019 count as multiple objects. See also: [Signer Lists and Reserves](signerlist.html#signer-lists-and-reserves).
- An [Owner Directory](directorynode.html) is a ledger entry that lists all objects related to an account, including all objects the account owns. However, the owner directory itself does not count towards the reserve.

### Looking Up Reserves

> Applications can look up the current base and incremental reserve values using the [server_info method][] or [server_state method][]:

Author: I would like to move this content out of this page entirely, but that's a slightly bigger task (#1709).
